src/views/Orders.vue

```diff
@@ -18,7 +18,7 @@
 
     <ion-content>
       <div v-if="segmentSelected === 'open'">
-        <div v-for="order in orders" :key="order.orderId" v-show="getShipGroups(order.items).length > 0">
+        <div v-for="order in orders" :key="order.groupValue" v-show="getShipGroups(order.items).length > 0">
```

Comment:
This should be extracted from orderId only

Reply:
Changed code to extract it with orderId